packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -165,6 +190,16 @@ export default baseMixins.extend<options>().extend({
 
   beforeMount () {
     this.hasWindow = typeof window !== 'undefined'
+
```

Comment:
These next 10 lines of code make my `v-navigation-drawer` which works with `expand-on-hover` and `mini-variant` throw consistent console errors 
```
index.js:245 Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at VueComponent.checkActivatorFixed (index.js:245)
    at VueComponent.updateDimensions (index.js:338)
```
for this file.

I have created a barebones [Codepen](https://codepen.io/kotoriii/pen/zYZWyXB) project to try and reproduce this outside of my local environment, but unfortunately the console error does not seem to appear there. Hence why I'm not opening a proper Issue Report on this, but raising this issue here instead. 

Perhaps the error can give you a hint of what might be wrong and if it also is / will affect other people

Reply:
@KaelWD @Tofandel